test/pg-cdc/mzcompose.yml

```diff
@@ -32,13 +32,13 @@ services:
     mzbuild: materialized
     command: --experimental --disable-telemetry
     ports:
-      - 6875
+      - 6875:6875
```

Comment:
[Publishing the Docker ports](https://docs.docker.com/config/containers/container-networking/#published-ports) makes it easier to debug `pg-cdc` failures.